tests/e2e/tests/regression/workbench/command-results.e2e.ts

```diff
@@ -134,3 +134,21 @@ test.before(async t => {
     // verify table view row count match with text view after client list command
     await workBenchActions.verifyClientListTableViewRowCount();
 });
+// https://redislabs.atlassian.net/browse/RI-4230
+test.only.before(async t => {
+    await acceptLicenseTermsAndAddDatabaseApi(ossStandaloneV5Config, ossStandaloneV5Config.databaseName);
+    await t.click(myRedisDatabasePage.workbenchButton);
+}).after(async() => {
+    await deleteStandaloneDatabaseApi(ossStandaloneV5Config);
+})('Verify that change screens when  capability not available - JSON,RediSearch', async t => {
```

Comment:
please add this test name to checklist
looks like it us 'Verify that user can see options on what can be done to work with capabilities in Workbench for docker'